### DISCUSSION TITLE
REMOVING ENV VARIABLES

### DISCUSSION THREAD

**Discussion by EdgarMtz00** [cBotFoundation/cBot.ts#76] [General]
Currently we receive configuration the following way: 
``` typescript
const mockCBootConfig: cBootConfig = {
    port: 7070,
    deploy: true,
    clientKey: process.env.DISCORD_BOT_TOKEN,
    clientId: process.env.DISCORD_CLIENT_ID,
    serverId: process.env.DISCORD_GUILD_ID,
    commands: [helloWorldCommand], // Fill with actual dummy Commands
    logger: myLogger
  }
```
Here, using environment variables is entirely up to the user. Since we do not request any specific environment variables elsewhere, I propose deleting the dotenv support and any associated .env file in the near future, preferably  once we setup a demo project apart from this one

**Comment by 54UL** [comment]
Go ahead, env configuration needs a rework, but we need still configuring stuff, let's start the configuration manager that reads files, when developing data base conectors this is a must

Concepts about the configutation manager (ideas)
- Multi profile (localhost, dev, prod, etc)
- json config files?
- simple config names such (db.json, bot.json)
- default config files


Or any other suggestion will be nicely recived
